go/install/install_darwin.go

```diff
@@ -17,6 +15,7 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/kardianos/osext"
+	kbnmInstaller "github.com/keybase/client/go/kbnm/installer"
```

Comment:
I don't think package aliases in go are meant to be all lowercase..  maybe `kbmni`?